src/main/resources/avro/alleleAnnotations.avdl

```diff
@@ -107,6 +107,22 @@ record VariantAnnotationSet {
 }
 
 /**
+A HGVSAnnotation record holds Human Genome Variation Society descriptions
+of the variant allele with respect to a transcript.
+*/
```

Comment:
A stupid question: are there any requirements about which of these fields can be null at the same time? I.e., can we have genomic, coding, or protein set, in any combination?